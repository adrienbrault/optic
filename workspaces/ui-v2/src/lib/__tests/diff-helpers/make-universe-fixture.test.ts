// import { makeUniverse } from './universes/makeUniverse';
//
// const universe_raw = require('./universes/simple-todo/universe.json');
//
// const universePromise = makeUniverse(universe_raw);
//
// test('simulated capture service can run in tests', async () => {
//   const { loadInteraction } = await universePromise;
//
//   const example = universe_raw.session.samples[0].uuid;
//   expect(await loadInteraction(example)).toMatchSnapshot();
// });
//
// test('simulated diff service can start a diff', async () => {
//   const { rawDiffs } = await universePromise;
//   expect(JSON.stringify(rawDiffs)).toMatchSnapshot();
// });
