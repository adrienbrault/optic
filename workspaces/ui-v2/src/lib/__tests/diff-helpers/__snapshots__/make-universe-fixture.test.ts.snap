// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simulated capture service can run in tests 1`] = `
Object {
  "interaction": Object {
    "request": Object {
      "body": Object {
        "contentType": "application/json",
        "value": Object {
          "asJsonString": null,
          "asText": null,
          "shapeHashV1Base64": "CAEaNQgAEgoKBHRhc2sSAggCEg0KB2R1ZURhdGUSAggCEgwKBmlzRG9uZRICCAQSCAoCaWQSAggCGiYIABIKCgR0YXNrEgIIAhIMCgZpc0RvbmUSAggEEggKAmlkEgIIAhomCAASCgoEdGFzaxICCAISDAoGaXNEb25lEgIIBBIICgJpZBICCAIaJggAEgoKBHRhc2sSAggCEgwKBmlzRG9uZRICCAQSCAoCaWQSAggCGiYIABIKCgR0YXNrEgIIAhIMCgZpc0RvbmUSAggEEggKAmlkEgIIAg==",
        },
      },
      "headers": Object {
        "asJsonString": null,
        "asText": null,
        "shapeHashV1Base64": null,
      },
      "host": "localhost",
      "method": "GET",
      "path": "/api/todos",
      "query": Object {
        "asJsonString": null,
        "asText": null,
        "shapeHashV1Base64": null,
      },
    },
    "response": Object {
      "body": Object {
        "contentType": "application/json",
        "value": Object {
          "asJsonString": null,
          "asText": null,
          "shapeHashV1Base64": "CAEaNQgAEgoKBHRhc2sSAggCEg0KB2R1ZURhdGUSAggCEgwKBmlzRG9uZRICCAQSCAoCaWQSAggCGiYIABIKCgR0YXNrEgIIAhIMCgZpc0RvbmUSAggEEggKAmlkEgIIAhomCAASCgoEdGFzaxICCAISDAoGaXNEb25lEgIIBBIICgJpZBICCAIaJggAEgoKBHRhc2sSAggCEgwKBmlzRG9uZRICCAQSCAoCaWQSAggCGiYIABIKCgR0YXNrEgIIAhIMCgZpc0RvbmUSAggEEggKAmlkEgIIAg==",
        },
      },
      "headers": Object {
        "asJsonString": null,
        "asText": null,
        "shapeHashV1Base64": null,
      },
      "statusCode": 200,
    },
    "tags": Array [],
    "uuid": "8eff95b1-4f6d-4d3c-ba71-5e3792780ce4",
  },
}
`;

exports[`simulated diff service can start a diff 1`] = `"[[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_gwQEFrHpO0\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"itemShapeId\\":\\"shape_f7gQgQ8p7G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_5GCvc8KB2p\\",\\"fieldShapeId\\":\\"shape_owJFnZQJeS\\"}}]}}}}},[\\"8eff95b1-4f6d-4d3c-ba71-5e3792780ce4\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_Zv48g7lL5e\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"itemShapeId\\":\\"shape_Fr2jskGj0G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_cOmYY7RoTV\\",\\"fieldShapeId\\":\\"shape_FVWIcOgFGF\\"}}]}}}}},[\\"8eff95b1-4f6d-4d3c-ba71-5e3792780ce4\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos/abcdfeg\\"}},{\\"Method\\":{\\"method\\":\\"GET\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"8eff95b1-4f6d-4d3c-ba71-5e37923780ce4\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"GET\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"8eff95b1-4f6d-4d3c-ba71-5e37923780ce4\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos/abcdfeg\\"}},{\\"Method\\":{\\"method\\":\\"GET\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"8eff95b1-4f6d-4d3c-ba71-5e37923780ce4\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"GET\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"8eff95b1-4f6d-4d3c-ba71-5e37923780ce4\\"]],[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_3kjV3YMXdP\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_FMLvgzBZRK\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_FMLvgzBZRK\\",\\"itemShapeId\\":\\"shape_M3jePX8HMF\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_oQxruSbdCD\\",\\"fieldShapeId\\":\\"shape_YtXLm2ULc3\\"}}]}}}}},[\\"fa0def9226-abf8-41a7-ba1f-7fd09c509090\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_dE2gzm1TWj\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_atTrTmH6j9\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_atTrTmH6j9\\",\\"itemShapeId\\":\\"shape_w9f5rUduX2\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_MxN4prqQaf\\",\\"fieldShapeId\\":\\"shape_5Htez4BFKO\\"}}]}}}}},[\\"fa0def9226-abf8-41a7-ba1f-7fd09c509090\\"]],[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_gwQEFrHpO0\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"itemShapeId\\":\\"shape_f7gQgQ8p7G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_5GCvc8KB2p\\",\\"fieldShapeId\\":\\"shape_owJFnZQJeS\\"}}]}}}}},[\\"1c0b722f-107c-4082-abe7-f3cdd1d92475\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_Zv48g7lL5e\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"itemShapeId\\":\\"shape_Fr2jskGj0G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_cOmYY7RoTV\\",\\"fieldShapeId\\":\\"shape_FVWIcOgFGF\\"}}]}}}}},[\\"1c0b722f-107c-4082-abe7-f3cdd1d92475\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos/5exvbk000\\"}},{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"2488a433-85f6-4b1f-bd80-23c2c46a5c87\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"2488a433-85f6-4b1f-bd80-23c2c46a5c87\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos/aa5exvbk000\\"}},{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"2488a433-85f6-4b1f-bd80-23c2c4ds6a5c87\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"2488a433-85f6-4b1f-bd80-23c2c4ds6a5c87\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos/5ebk000\\"}},{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"2488a433-85f6-4b1fdsa-bd80-23c2c46a5c87\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"2488a433-85f6-4b1fdsa-bd80-23c2c46a5c87\\"]],[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_gwQEFrHpO0\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"itemShapeId\\":\\"shape_f7gQgQ8p7G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_5GCvc8KB2p\\",\\"fieldShapeId\\":\\"shape_owJFnZQJeS\\"}}]}}}}},[\\"1c47baf2-0b1d-43a1-a5c5-64f9249801a5\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_Zv48g7lL5e\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"itemShapeId\\":\\"shape_Fr2jskGj0G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_cOmYY7RoTV\\",\\"fieldShapeId\\":\\"shape_FVWIcOgFGF\\"}}]}}}}},[\\"1c47baf2-0b1d-43a1-a5c5-64f9249801a5\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos\\"}},{\\"Method\\":{\\"method\\":\\"POST\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_UOIsxzICu5\\"}}}},[\\"a5eaf726-5548-4e37-8bc8-d5fb2accb2e9\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"POST\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_UOIsxzICu5\\"}}}},[\\"a5eaf726-5548-4e37-8bc8-d5fb2accb2e9\\"]],[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_gwQEFrHpO0\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"itemShapeId\\":\\"shape_f7gQgQ8p7G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_5GCvc8KB2p\\",\\"fieldShapeId\\":\\"shape_owJFnZQJeS\\"}}]}}}}},[\\"e930e3f5-a006-46d8-85ae-447ce3150906\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_Zv48g7lL5e\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"itemShapeId\\":\\"shape_Fr2jskGj0G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_cOmYY7RoTV\\",\\"fieldShapeId\\":\\"shape_FVWIcOgFGF\\"}}]}}}}},[\\"e930e3f5-a006-46d8-85ae-447ce3150906\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/todos/5exvbk000\\"}},{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"75f97fe4-4b0c-47e3-834f-ba991dd525a7\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"PATCH\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_it2OyjUysW\\"}}}},[\\"75f97fe4-4b0c-47e3-834f-ba991dd525a7\\"]],[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_gwQEFrHpO0\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_cEkQAVQ3ib\\",\\"itemShapeId\\":\\"shape_f7gQgQ8p7G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_5GCvc8KB2p\\",\\"fieldShapeId\\":\\"shape_owJFnZQJeS\\"}}]}}}}},[\\"a3af782f-8e44-4e17-806d-025c8fb0a4fd\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_Zv48g7lL5e\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_0xeeapZ7UZ\\",\\"itemShapeId\\":\\"shape_Fr2jskGj0G\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_cOmYY7RoTV\\",\\"fieldShapeId\\":\\"shape_FVWIcOgFGF\\"}}]}}}}},[\\"a3af782f-8e44-4e17-806d-025c8fb0a4fd\\"]],[{\\"UnmatchedRequestBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Url\\":{\\"path\\":\\"/api/lists/list123\\"}},{\\"Method\\":{\\"method\\":\\"GET\\"}},{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_AsEexQkVwC\\"}}}},[\\"a3af782f-8e44-4e17-806d-032323fb0a4fd\\"]],[{\\"UnmatchedResponseBodyContentType\\":{\\"interactionTrail\\":{\\"path\\":[{\\"Method\\":{\\"method\\":\\"GET\\"}},{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecPath\\":{\\"pathId\\":\\"path_AsEexQkVwC\\"}}}},[\\"a3af782f-8e44-4e17-806d-032323fb0a4fd\\"]],[{\\"UnmatchedRequestBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"RequestBody\\":{\\"contentType\\":\\"application/json\\"}}]},\\"requestsTrail\\":{\\"SpecRequestBody\\":{\\"requestId\\":\\"request_SqY61Qc9Mi\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_Lx1MrhWlFb\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_Lx1MrhWlFb\\",\\"itemShapeId\\":\\"shape_QU1rtECeM2\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_eHl286agXw\\",\\"fieldShapeId\\":\\"shape_PTjiKnpFzQ\\"}}]}}}}},[\\"a3af782f-8e44-4e1327-806d-032323fb0a4fd\\"]],[{\\"UnmatchedResponseBodyShape\\":{\\"interactionTrail\\":{\\"path\\":[{\\"ResponseBody\\":{\\"contentType\\":\\"application/json\\",\\"statusCode\\":200}}]},\\"requestsTrail\\":{\\"SpecResponseBody\\":{\\"responseId\\":\\"response_RkkvxIt2RG\\"}},\\"shapeDiffResult\\":{\\"UnmatchedShape\\":{\\"jsonTrail\\":{\\"path\\":[{\\"JsonArrayItem\\":{\\"index\\":1}},{\\"JsonObjectKey\\":{\\"key\\":\\"dueDate\\"}}]},\\"shapeTrail\\":{\\"rootShapeId\\":\\"shape_ToF242uYVA\\",\\"path\\":[{\\"ListItemTrail\\":{\\"listShapeId\\":\\"shape_ToF242uYVA\\",\\"itemShapeId\\":\\"shape_ohd8yFyzEg\\"}},{\\"ObjectFieldTrail\\":{\\"fieldId\\":\\"field_TxVlnhtLaa\\",\\"fieldShapeId\\":\\"shape_3Xt9wp5UxL\\"}}]}}}}},[\\"a3af782f-8e44-4e1327-806d-032323fb0a4fd\\"]]]"`;
